src/lib/paste.ts

```diff
@@ -39,7 +48,46 @@ const removeIDs = (slice: ManuscriptSlice) => {
   })
 }
 
-export const transformPasted = (slice: ManuscriptSlice): ManuscriptSlice => {
+const pasteSection = (slice: ManuscriptSlice, view: ManuscriptEditorView) => {
+  const { state, dispatch } = view
+  let sectionTitleNode: ManuscriptNode | null = null
+  const sectionBodyNodes: ManuscriptNode[] = []
+  ClipboardEvent
+
+  slice.content.descendants((node) => {
+    if (node.type === node.type.schema.nodes.section) {
+      node.content.forEach((item) => {
+        const sectionBodyNode = state.schema.nodes.paragraph.create(
+          {},
+          item.content
+        ) as ParagraphNode
```

Comment:
the content could be anything figure,table...

Reply:
I was trying to fix the specific case mentioned in https://jira.atypon.com/browse/LEAN-3823. I've also reviewed other scenarios and noticed that the editor currently doesn't support pasting tables and figures. To fully support these elements, I believe additional tickets should be created to handle pasting different types of content, which can then be integrated within sections.